util/perl/OpenSSL/paramnames.pm

```diff
@@ -94,6 +94,7 @@ my %params = (
     'ALG_PARAM_MAC' =>          "mac",          # utf8_string
     'ALG_PARAM_PROPERTIES' =>   "properties",   # utf8_string
     'ALG_PARAM_FIPS_APPROVED_INDICATOR' => 'fips-indicator',   # int, -1, 0 or 1
+    'ALG_PARAM_SECURITY_CATEGORY' => "security-category", # int, 0 .. 5
```

Comment:
-1..5?

Reply:
I think 0..5 is correct here.  The parameter cannot ever be -1.  That's the error return from the access function.